conf/application.conf

```diff
@@ -176,88 +176,97 @@ external-url {
 }
 
 services-config {
-  list = "cds,atar,gagmr,route1,ctc,gbss,ars,ivd,ndrc"
-
-  cds {
-    enrolment = "HMRC-CUS-ORG"
-    shortName = "CDS"
-    callBack = "https://www.gov.uk/guidance/get-access-to-the-customs-declaration-service#after-youve-subscribed"
-    accessibility = "http://localhost:12346/accessibility-statement/get-access-customs-declaration-service"
-    friendlyName = "the_Customs_Declaration_Service"
-    friendlyNameWelsh = "y_Gwasanaeth_Datganiad_Tollau"
-  }
+  list = "ars,atar,cds,ctc,esc,gagmr,gbss,ivd,ndrc,route1"
+
+    ars {
+        enrolment = "HMRC-ATAR-ORG"
+        shortName = "ARS"
+        callBack = "http://localhost:12600/advance-valuation-ruling/applications-and-rulings",
+        accessibility = "http://localhost:12346/accessibility-statement/advance-valuation-ruling",
+        friendlyName = "advance_rulings"
+        friendlyNameWelsh = "ddyfarniadau_uwch"
+    }
 
-  gbss {
-    enrolment = "HMRC-SS-ORG"
-    shortName = "S&S_GB"
-    callBack = "http://localhost:9582/safety-and-security-subscription/successfully-enrolled"
-    accessibility = "http://localhost:12346/accessibility-statement/safety-and-security-subscription"
-    friendlyName = "the_Safety_and_Security_GB_service"
-    friendlyNameWelsh = "y_gwasanaeth_Sicrwydd_a_Diogelwch_(S&S_GB)"
-  }
+    atar {
+        enrolment = "HMRC-ATAR-ORG",
```

Comment:
these ones are comma separated but the rest aren't - is that an issue?

Reply:
I'll remove them, it shouldn't be a problem either way